source/modules/Git.rst

```diff
@@ -101,8 +101,8 @@ Git uses shortcuts to address *remotes*. When you *clone* a repository from its
 .. code-block:: bash
 
   $ git remote -v
-  origin  https://github.com/UWPCE-PythonCert-ClassRepos/IntroToPython.git (fetch)
-  origin  https://github.com/UWPCE-PythonCert-ClassRepos/IntroToPython.git (push)
+  origin  https://github.com/UWPCE-PythonCert-ClassRepos/ExampleRepo.git (fetch)
```

Comment:
Should we use “YouClassRepoNameHere” instead?

Reply:
since this was before `YouClassRepoNameHere` was even mentioned as that is something that is going to be used throughout as examples.. i decided to use generic example repo name.

Reply:
Makes sense. I'll merge.

I think in the next go-around, we should make a copy of this doc for each class, and then search+replace the actual class repo name, so students can copy and paste away, and there will be much less confusion.